### PR TITLE
add support for E.get()

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
related: #2502

@michaelfig  Is there a good way to mark `E.G()` as deprecated?

### CODE REVIEW

File: packages/eventual-send/src/index.d.ts [hunk 1]
```diff
@@ -99,6 +99,17 @@ interface EProxy {
    */
   readonly G<T>(x: T): ESingleGet<Unpromise<T>>;
```
Comment (michaelfig) [inline packages/eventual-send/src/index.d.ts:94]: You may be able to mark the above with `@deprecated`.

**Review by michaelfig — CHANGES_REQUESTED** [review]
Would you also consider adding a global search-and-replace for `\bE\.G\b` to `E.get`?

That way it's only dapps and external users who might possibly have (deprecated) references to `E.G`.

File: packages/eventual-send/src/index.d.ts (hunk 1 above)
Comment (michaelfig) [reply packages/eventual-send/src/index.d.ts:94]: I did that, but it doesn't seem to change anything.  Unfortunate.

**Review by michaelfig — APPROVED** [review]
LGTM, but I don't want to approve my own changes.  @Chris-Hibbert, @erights can you have a quick peek?

**Comment by michaelfig** [comment]
> Would you also consider adding a global search-and-replace for `\bE\.G\b` to `E.get`?

I actually used `\bE\.G\b(\(|;)` to `E.get$1`.  Seemed to work well.

**Review by Chris-Hibbert — COMMENTED** [review]
Looks great!  I couldn't find any missed instances locally.  Thanks very much.

**Pull request merged**